internal/pbio/pbio_test.go

```diff
@@ -0,0 +1,145 @@
+// Protocol Buffers for Go with Gadgets
```

Comment:
Content of this file is (partially) copy-pasted from gogo/io/io_test.go

Only needed tests (varint-only).